### PR TITLE
Fix tutorials

### PR DESCRIPTION
**Pull request by akorgor**
This PR contains minor fixes of the tutorials, changes the default neuron model for better comparability and adds the online evidence accumulation task.

### CODE REVIEW

File: pynest/examples/eprop_plasticity/eprop_supervised_regression_infinite-loop.py [hunk 1]
```diff
@@ -399,7 +397,7 @@
 # %% ###########################################################################################################
 # Create output
 # ~~~~~~~~~~~~~
-# Then, we load the x and y values of an image of the word "chaos" written by hand and construct a roughly
+# Then, we load the x and y values of an image of an infinite loop and construct a roughly
```
Comment (JesusEV) [inline pynest/examples/eprop_plasticity/eprop_supervised_regression_infinite-loop.py]: Doesn't the concept of an "infinite loop" remind of structures like `for( ; ; ) { }` or `while(true) { }` , or even worse, a closed curve that is infinite? Maybe it is clearer if we refer to it as a "lemniscate". This nomenclature could avoid confusions.

File: pynest/examples/eprop_plasticity/online_eprop_supervised_classification_evidence-accumulation.py [hunk 2]
```diff
@@ -0,0 +1,815 @@
...
+n_out = 2  # number of readout neurons
+
+model_nrn_reg = "eprop_iaf"
+
+params_nrn_out = {
+    "C_m": 1.0,  # pF, membrane capacitance - takes effect only if neurons get current input (here not the case)
+    "E_L": 0.0,  # mV, leak / resting membrane potential
+    "eprop_isi_trace_cutoff": 10,  # cutoff of integration of eprop trace between spikes
+    "I_e": 0.0,  # pA, external current input
+    "loss": "mean_squared_error",  # loss function
```
Comment (JesusEV) [inline pynest/examples/eprop_plasticity/online_eprop_supervised_classification_evidence-accumulation.py:198]:
```suggestion
```

**Review by JesusEV — CHANGES_REQUESTED** [review]

File: pynest/examples/eprop_plasticity/eprop_supervised_regression_infinite-loop.py (hunk 1 above)
Comment (akorgor) [reply pynest/examples/eprop_plasticity/eprop_supervised_regression_infinite-loop.py]: Sure, good suggestion! I myself was never happy with that vague name. I will change it

File: pynest/examples/eprop_plasticity/online_eprop_supervised_classification_evidence-accumulation.py (hunk 2 above)
Comment (akorgor) [inline pynest/examples/eprop_plasticity/online_eprop_supervised_classification_evidence-accumulation.py:198]: Could you please elaborate on this change? The script would work with your change as `"means_squared_error"` is the default, but wouldn't it be better to make it explicit?

Comment (JesusEV) [reply pynest/examples/eprop_plasticity/online_eprop_supervised_classification_evidence-accumulation.py:198]: Hold on, doesn't complain if you include that line?

Please check:
9891267b6cb0258417b830e5f1dadd7ea66c03c8

Comment (JesusEV) [reply pynest/examples/eprop_plasticity/online_eprop_supervised_classification_evidence-accumulation.py:198]: That line appears in the other online_* files. I overlooked that. 

Going back: as for the new models the `loss` variable can assume only one single value I think it make sense that `loss` is not a parameter that can be set.

**Review by JesusEV — APPROVED** [review]

File: pynest/examples/eprop_plasticity/online_eprop_supervised_classification_evidence-accumulation.py (hunk 2 above)
Comment (JesusEV) [reply pynest/examples/eprop_plasticity/online_eprop_supervised_classification_evidence-accumulation.py:198]: I suggest my suggested change is not added to this PR but the change is added or discussed separately.

**Review by JesusEV — APPROVED** [review]

**Review by JesusEV — APPROVED** [review]

File: pynest/examples/eprop_plasticity/online_eprop_supervised_classification_evidence-accumulation.py (hunk 2 above)
Comment (akorgor) [reply pynest/examples/eprop_plasticity/online_eprop_supervised_classification_evidence-accumulation.py:198]: I agree, we can remove the `loss` parameter.

**Pull request merged**